cmd/check.go

```diff
@@ -46,18 +56,13 @@ var checkCmd = &cobra.Command{
 	},
 }
 
-func init() {
-	rootCmd.AddCommand(checkCmd)
-}
-
 func validateArgs(cmd *cobra.Command, args []string) error {
-	if len(args) != 1 {
-		return fmt.Errorf("you must specify a spec file")
-	}
+	if len(args) != 0 {
+		_, err := os.Stat(args[0])
 
-	_, err := os.Stat(args[0])
-	if os.IsNotExist(err) {
-		return fmt.Errorf("spec file %s does not exist", args[0])
+		if os.IsNotExist(err) {
+			return fmt.Errorf("spec file %s does not exist", args[0])
```

Comment:
I think this would be clearer to the end-user since we are calling the flag bundle
```suggestion
			return fmt.Errorf("bundle file %s does not exist", args[0])
```